src/js/gi/components/AboutYourSchoolFields.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import If from './If';
+import InstitutionNameInput from './InstitutionNameInput';
+
+class AboutYourSchoolFields extends React.Component {
+
+  render() {
+    return (
+      <span>
+        <a name="about_your_school"></a>
+        <div id="institution-search-form" className="form-group">
+
+          <If condition={this.props.labels}>
+            <div id="online-classes-form" className="form-group">
+              <label>Will you be taking classes online? </label>
+
+              <input type="radio" id="online-classes-all"
+                  name="online_classes" value="yes"
+                  defaultChecked={this.props.onlineClasses && this.props.onlineClasses === 'all'}/>
+              <label className="radio" htmlFor="online-classes-all" style={{ display: 'inline-block' }}>All</label>
```

Comment:
Turns out `inline-block` is needed so I moved it into a stylesheet. (These are old styles; I've spent as little time as possible cleaning up old css.)